src/VCR/Request.php

```diff
@@ -75,6 +75,13 @@ public function matches(Request $request, array $requestMatchers)
         return true;
     }
 
+    public function collectMismatches(Request $request, array $requestMatchers, MismatchExplainer $mismatchExplainer) {
+        // convert simple callables here to CallableRequestMatcher
```

Comment:
I was thinking of doing that way in the beginning when calling `Configuration::addRequestMatcher`. So here `collectMismatches` would get a list of RequestMatcher objects which have a `match` and a `getMismatches` method.